1.16.1/SNP_DirectIP/src/DirectIP/Settings.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+class Settings
+{
+public:
+	Settings() {};
+	virtual ~Settings() {};
+
+	virtual void init() = 0;
+	virtual void release() = 0;
+
+	virtual const char* getHostIPString() = 0;
+	virtual const char* getHostPortString() = 0;
```

Comment:
Maybe `getHostPort` could return the result of `std::stoul` instead of calling `atoi` above.

Reply:
Yes, this can be done already in the function. Will update.